posydon/grids/psygrid.py

```diff
@@ -891,12 +886,11 @@ def decide_columns(key_in_config, defaults):
                     if self.config["accept_missing_profile"]:
                         warnings.warn("Including MESA run despite the missing "
                                       "profile in {}\n".format(run.path))
-                        ignore_data = False
+                        ignore.reason = None
```

Comment:
Should we may better remove this line, because it is not doing anything, while my confusing. It will only reset ignore to None, if it was None before.

Reply:
It actually does - you can always reset the ignore reason to `None`, you just can't set it to a lower-order ignore reason. So this does have an effect.

Reply:
That's true. Here I'm more questioning, why this line is there at all because it is behind a `if not ignore`
Hence for me the effective code reads:
```
if ignore.reason == None:
    ignore.reason == None
```

Reply:
I see what you mean. I guess I put it there for emphasis. But it's OK now, I removed it.